2023/12/02 13:15:44 failed to dial tcp connection, retrying
2023/12/02 13:15:45 failed to dial tcp connection, retrying
[2023-12-02T13:15:45.104Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-12-02T13:15:45.105Z] info  Using user-data-dir ~/.local/share/code-server
[2023-12-02T13:15:45.124Z] info  Using config file ~/.config/code-server/config.yaml
[2023-12-02T13:15:45.124Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-12-02T13:15:45.124Z] info    - Authentication is disabled
[2023-12-02T13:15:45.124Z] info    - Not serving HTTPS
====================================>token: 4946b095-7c5f-43bb-80fb-b4ffd9ec879e
[13:15:47] 




[13:15:47] Extension host agent started.
[13:15:49] [127.0.0.1][1cf082f5][ManagementConnection] New connection established.
[13:15:50] [127.0.0.1][1e1f4afb][ExtensionHostConnection] New connection established.
[13:15:50] [127.0.0.1][1e1f4afb][ExtensionHostConnection] <391> Launched Extension Host Process.
[13:15:54] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[2023-12-02T13:18:00.378Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
[13:18:14] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[16:46:44] [127.0.0.1][1cf082f5][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[16:46:44] [127.0.0.1][1e1f4afb][ExtensionHostConnection] <391> Extension Host Process exited with code: 0, signal: null.
2023/12/02 16:56:46 exit program after timeout
